rixs/tests/test_process2d.py

```diff
@@ -0,0 +1,93 @@
+from rixs import process2d
+import numpy as np
+from numpy.random import rand
+from numpy.testing import assert_array_almost_equal
+
+
+def make_fake_image(curvature, elastic_y, sigma=2, noise=0.002):
+    """Make a fake list of photon events.
+    Parameters
```

Comment:
I think we are missing a blank line here

Reply:
Will remove